ios/RNTrackPlayer/Models/Track.swift

```diff
@@ -42,6 +43,7 @@ class Track: NSObject {
         self.album = dictionary["album"] as? String
         self.genre = dictionary["genre"] as? String
         self.desc = dictionary["description"] as? String
+        self.pitchAlgorithm = dictionary["pitchAlgorithm"] as? String
```

Comment:
For this you can probably export a constant that the JS side can use. Or you can add a helper on the JS side to make sure that only the allowed values are passed in

Reply:
I've added constants that are exported. 

(Just for the record)